Source/UI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -73,10 +73,10 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
         style.cells.forEach { type in
             let isMandatory = type.style?.isMandatory ?? true
             let value = type.getText(from: data)
-            let hasValue = value != nil
+            let isEmpty = value?.isEmpty ?? true
             country = data?.address?.country
-            textValueOfCellType[type.index] = hasValue ? value : (!isMandatory ? "" : nil)
-            errorFlagOfCellType[type.index] = hasValue ? false : (value != nil ? isMandatory : nil)
+            textValueOfCellType[type.index] = !isEmpty ? value : (!isMandatory ? "" : nil)
+            errorFlagOfCellType[type.index] = !isEmpty ? false : (value != nil ? isMandatory : nil)
```

Comment:
```suggestion
let hasValue = value?.isEmpty == false
country = data?.address?.country
textValueOfCellType[type.index] = hasValue ? value : (!isMandatory ? "" : nil)
errorFlagOfCellType[type.index] = hasValue ? false : (value != nil ? isMandatory : nil)
```

Reason for this suggestion is to avoid the `!isEmpty` for the comparison. Can also achieve by swapping the last 2 parts in the ternary operator and using `isEmpty`. Just thinking `hasValue` sounded better, but up to you how you'd prefer to avoid the syntax.

Reply:
I would like to keep it `isEmpty` like `value?.isEmpty`.

Reply:
I was not suggesting to remove `isEmpty` from the string check, I was suggesting to not use ternary operator with a negation.

```swift
// Current
!bool ? value1 : value2

// Suggested
bool ? value2 : value1
```

Ternary operator is great for being concise, but it is a complex line to understand. Booleans are easy to get wrong. I'd like us to keep it simple and don't use ternary operator while negating booleans.

If you review my code suggestion, I have not suggested we don't use `value?.isEmpty`, I was merely suggesting that you inverse the Boolean logic to avoid the negation on your assertion result